internal/fsutil/path.go

```diff
@@ -0,0 +1,47 @@
+package fsutil
+
+import (
+	"path/filepath"
+	"strings"
+)
+
+// These functions exists because we work with slash terminated directory paths
+// that come from deb package tarballs but standard library path functions
+// treat slash terminated paths as unclean.
+
+// Like filepath.Dir() but trailing slash on input is ignored and the return
+// value always includes trailing slash.
```

Comment:
Function comments should start with the name of the function e.g. `// Dir cleans the filepath and then returns all but the last element of the path. It includes a trailing slash.`

Reply:
Updated the comment to start with function name, but kept the description in terms of `filepath.Dir()` so that it's obvious why it exists and how is it different.